level1/cmds/dmode.asm

```diff
@@ -44,13 +42,9 @@
 * May be a back-port from Carl Kreider's dmode for OSK
 *
 * 2 / 4    2004/02/28  Rodney V. Hamilton
-* Disassembled new dmode, added comments and KD/RAK mods.
```

Comment:
This is a real comment.

Reply:
Good point. I forced push a new commit that only removes $Id$.